src/Nethermind/Nethermind.Runner/Ethereum/Steps/StartRpc.cs

```diff
@@ -30,8 +32,34 @@ public StartRpc(INethermindApi api)
         public async Task Execute(CancellationToken cancellationToken)
         {
             IJsonRpcConfig jsonRpcConfig = _api.Config<IJsonRpcConfig>();
+            IKeyStoreConfig keyStoreConfig = _api.Config<IKeyStoreConfig>();
             ILogger logger = _api.LogManager.GetClassLogger();
 
+
+            // Update the JWT secret path based on the data directory.
+            try
+            {
+                string defaultPath = "keystore/jwt-secret";
+                string newPath = Path.Combine(keyStoreConfig.KeyStoreDirectory, "jwt-secret");
```

Comment:
We don't need all power of `Path.Combine` here:
```suggestion
                string newPath = Path.Join(keyStoreConfig.KeyStoreDirectory, "jwt-secret");
```